Sources/ClientRuntime/Networking/Http/HttpClientConfiguration.swift

```diff
@@ -45,7 +45,7 @@ public class HttpClientConfiguration {
     ///   - protocolType: The HTTP scheme (`http` or `https`) to be used for API requests.  Defaults to the operation's standard configuration.
     public init(
         connectTimeout: TimeInterval? = nil,
-        socketTimeout: TimeInterval? = nil,
+        socketTimeout: TimeInterval? = 60.0,
```

Comment:
If we are providing a non-nil default, should the param be non-optional?

Reply:
Changed as suggested.